requestmanager/requestmanager_test.go

```diff
@@ -212,9 +212,63 @@ func TestCancelRequestInProgress(t *testing.T) {
 
 	errors := testutil.CollectErrors(requestCtx, t, returnedErrorChan1)
 	require.Len(t, errors, 1)
-	_, ok := errors[0].(graphsync.RequestContextCancelledErr)
+	_, ok := errors[0].(graphsync.RequestClientCancelledErr)
 	require.True(t, ok)
 }
+func TestCancelRequestImperativeNoMoreBlocks(t *testing.T) {
+	ctx := context.Background()
+	td := newTestData(ctx, t)
+	requestCtx, cancel := context.WithTimeout(ctx, time.Second)
```

Comment:
thinking out loud: for tests running on the CI a second timeout _might_ be aggressive.

Reply:
if it's cool let's address when we convert to unified CI - -from what I've seen those issues come up a lot more with the Unified CI that runs on all platforms. I suspect that will lead to a clarifying effort.

Reply:
Of course. timeout increase may be a total YAGNI anyway